### ISSUE TITLE
Deserializing fails on float values with "SizeSpecification"

### ISSUE THREAD

**Issue by averon420** [recyclarr/recyclarr#219]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Custom formats (in sonarr) with float values in the "SizeSpecification" lead to `[ERR] HTTP error: Response could not be deserialized to JSON: GET http://REDACTED:8989/api/v3/customformat` error. It doesn't matter if the CF is synced from guide, or already existing in sonarr.

### Expected Behavior

Expected to use float values, since Sonarr can also handle it.

### Steps To Reproduce

1. Import the custom format below
```
{
  "name": "Double Example for Size",
  "includeCustomFormatWhenRenaming": false,
  "specifications": [
    {
      "name": "Size",
      "implementation": "SizeSpecification",
      "negate": false,
      "required": false,
      "fields": {
        "min": 0.1,
        "max": 0.9
      }
    }
  ]
}
```
2. Run `recyclarr sync sonarr `
3. Get error
4. Change the min value to a whole number, repeat the command of step 2. and get no error

### Environment & Versions

```bash
- Operating System: Ubuntu/Docker
- Architecture: x64
- Recyclarr Version: 6.0.2
- Recyclarr Build: lscr.io/linuxserver/sonarr:develop
- Sonarr Version: 4.0.0.731
```

### Recyclarr Log File(s)

not needed

### Configuration YAML

```YAML
not needed
```

**Comment by glassbase** [comment]
Maybe my issue is same as this issue but with Radarr https://github.com/recyclarr/recyclarr/issues/227

**Comment by rcdailey** [comment]
Yeah, same issue as 227. I'm going to close this one as a duplicate. I appreciate your bug report. Note that this has always been documented as a [known issue](https://recyclarr.dev/wiki/troubleshooting/known-issues/); I don't have any immediate plans to address this since the guides do not use this specification.

**Comment by rcdailey** [comment]
I closed the wrong one... sorry about that.

**Comment by rcdailey** [comment]
@BlackDark I deleted your comment mainly because I'm not OK with "advertising" here.
